src/main/java/fr/scrumtogether/scrumtogetherapi/exceptions/TeamException.java

```diff
@@ -0,0 +1,12 @@
+package fr.scrumtogether.scrumtogetherapi.exceptions;
+
+public class TeamException extends RuntimeException {
```

Comment:
J'aurais plutôt fait une EntityCreationException.
```suggestion
public class EntityCreationException extends RuntimeException {
```